From 6153b0a580420af982ed28d0af31a43aeaf759f7 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 22 Sep 2015 15:09:35 +0800
Subject: [PATCH 0767/1691] MLK-11407-6: ARM: imx: mach-imx7d: remove
 mxc_restart

commit 6153b0a580420af982ed28d0af31a43aeaf759f7 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Use wdog driver to reset system instead of mxc_restart().

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/mach-imx/mach-imx7d.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-imx/mach-imx7d.c b/arch/arm/mach-imx/mach-imx7d.c
index 98f7ee3..d430456 100644
--- a/arch/arm/mach-imx/mach-imx7d.c
+++ b/arch/arm/mach-imx/mach-imx7d.c
@@ -139,5 +139,4 @@ DT_MACHINE_START(IMX7D, "Freescale i.MX7 Dual (Device Tree)")
 	.init_machine	= imx7d_init_machine,
 	.init_late	= imx7d_init_late,
 	.dt_compat	= imx7d_dt_compat,
-	.restart	= mxc_restart,
 MACHINE_END
-- 
1.9.1

